src/DSharp.Mscorlib/Array.cs

```diff
@@ -78,26 +95,35 @@ public sealed class Array : ICollection
 
         public extern object ReduceRight(ArrayItemReduceCallback callback, object initialValue);
 
+        // This method is static in the CLR -> void Reverse(Array array)
         public extern void Reverse();
 
+        // Shift method is not part of CLR. We might want to have it into an abstraction class.
         public extern object Shift();
 
+        // Slice methods are not part of CLR. We might want to have them into an abstraction class.
         public extern Array Slice(int start);
 
         public extern Array Slice(int start, int end);
 
+        // This some methods are not part of CLR. As far as I know, they're not even in Linq as Linq implements something different (Any ?)
         public extern bool Some(ArrayFilterCallback filterCallback);
 
         public extern bool Some(ArrayItemFilterCallback itemFilterCallback);
 
+        // All of the Sort methods in CLR are static.
+        // This should be -> static void Sort(Array array)
         public extern void Sort();
```

Comment:
This should be deprecated as well.

Reply:
Shall I use [Obsolete("some reason", true)]? Or we don't want the error bool to be true in these cases?

Reply:
I think if we go with `error: true` then we need to remove the API completely

Reply:
`error: true` is being used when there is an interface that forces us to have a method there, but it doesn't work on DSharp